applications/FemToDemApplication/custom_python/fem_to_dem_python_application.cpp

```diff
@@ -82,6 +82,7 @@ namespace Python
 	KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,INTEGRATION_COEFFICIENT);
 	KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,MAPPING_PROCEDURE);
 	KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,IS_DEM);
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,IS_SKIN);

```

Comment:
Why not using the flag BOUNDARY?. The skinf_detection_process already applies it

Reply:
Because I didn't know hehe let me try it. thx!!

Reply:
I recommend to look for flags instead of creating new bool variable